research/src/pages/documentation/get-involved.mdx

```diff
@@ -163,10 +158,9 @@ Once you have sign off on the spec from these stakeholders, you can advance your
 
 ### 3.1 Determine outcome: Open UI Component, Graduated Spec, or Both
 
-The first step of Stage 3 is to determine if this feature is going to be added to the platform directly by browser engine implementers, or implemented in JavaScript, HTML and CSS as a web Component.
-For implementation in a web browser, it's often the case that you might first implement a web component. So often the answer may be both.
+The first step of Stage 3 is to determine if this feature is going to be added to the platform directly by browser engine implementers, or implemented in JavaScript, HTML and CSS as a web Component. For implementation in a web browser, it's often the case that you might first implement a web component. So often the answer may be both.
```

Comment:
maybe also update “web Component” to be “Web Component”?

Reply:
and update the later mention of “web component” to also be “Web Component” for consistency?